internal/services/recoveryservices/recovery_services_vault_resource.go

```diff
@@ -632,6 +655,16 @@ func resourceRecoveryServicesVaultRead(d *pluginsdk.ResourceData, meta interface
 		d.Set("encryption", []interface{}{encryption})
 	}
 
+	vaultSettingsId := replicationvaultsetting.NewReplicationVaultSettingID(id.SubscriptionId, id.ResourceGroupName, id.VaultName, "default")
+	vaultSetting, err := vaultSettingsClient.Get(ctx, vaultSettingsId)
+	if err != nil {
+		return fmt.Errorf("reading Recovery Service Vault Setting %s: %+v", id.String(), err)
+	}
+	if vaultSetting.Model != nil && vaultSetting.Model.Properties != nil && vaultSetting.Model.Properties.VMwareToAzureProviderType != nil {
+		providerType := vaultSetting.Model.Properties.VMwareToAzureProviderType
+		d.Set("classic_vmware_replication_enabled", strings.EqualFold(*providerType, "vmware"))
+	}
```

Comment:
Can we break these conditions up for readability
```suggestion
	if vaultSetting.Model != nil && vaultSetting.Model.Properties != nil {
		if v := vaultSetting.Model.Properties.VMwareToAzureProviderType; v != nil {
			d.Set("classic_vmware_replication_enabled", strings.EqualFold(*v, "vmware"))
		}
	}
```